src/components/Header/styles.js

```diff
@@ -0,0 +1,43 @@
+/* (C) Copyright 2019 Hewlett Packard Enterprise Development LP. */
```

Comment:
Actually this whole file is not being imported anymore in header index I think. we. can delete

Reply:
can do